spark/src/main/java/com/netflix/iceberg/spark/source/Reader.java

```diff
@@ -260,26 +277,50 @@ private Schema lazyExpectedSchema() {
   }
 
   private static class TaskDataReader implements InputPartitionReader<InternalRow> {
+    private final class DecryptedFileScanTask {
```

Comment:
Is this still used?

Reply:
lol yea you're right. thanks for catching that!